helm/cluster-api-provider-cloud-director-app/templates/certificate.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: infrastructure-vcd
+    {{- include "labels.common" . | nindent 4 }}
+  name: capvcd-serving-cert
+  namespace: {{ .Release.Namespace }}
+spec:
+  dnsNames:
+  - capvcd-webhook-service.{{ .Release.Namespace }}.svc
+  - capvcd-webhook-service.{{ .Release.Namespace }}.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: capvcd-selfsigned-issuer # <= FILECHECK: SHOULD WE USE selfsigned-giantswarm?

```

Comment:
We could use the giantswarm default issuer IIRC

Reply:
we could, but this keeps the chart closer to upstream